### PR TITLE
Fixes missing dependencies in the build

### PR DESCRIPTION
**Pull request by adamretter**
Closes https://github.com/xspec/xspec-maven-plugin-1/issues/73

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -79,12 +79,27 @@
       <artifactId>xmlresolver</artifactId>
       <version>4.5.1</version>
     </dependency>
+    <dependency>
```
Comment (cmarchand) [inline pom.xml:82]: Why did you add this dependency ?

File: pom.xml [hunk 2]
```diff
@@ -79,12 +79,27 @@
...
+      <artifactId>commons-io</artifactId>
+      <version>2.17.0</version>
+    </dependency>
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
       <version>4.13.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```
Comment (cmarchand) [inline pom.xml:98]: And this one ?

**Review by cmarchand — COMMENTED** [review]
Two questions about added dependencies...

File: pom.xml (hunk 1 above)
Comment (adamretter) [reply pom.xml:82]: Because according to running `dependency:analyze-only` it is in use in the code base, which I suspect is correct ;-)

File: pom.xml (hunk 2 above)
Comment (adamretter) [reply pom.xml:98]: It is used in the code base here: https://github.com/xspec/xspec-maven-plugin-1/blob/master/src/main/java/io/xspec/maven/xspecMavenPlugin/utils/CatalogWriter.java#L88

File: pom.xml (hunk 1 above)
Comment (cmarchand) [reply pom.xml:82]: It is.

but `org.eclipse.sisu:org.eclipse.sisu.plexus` is included by `org.apache.maven:maven-plugin-api` and by `org.apache.maven:maven-core`, and I have dependencies declared on this ones. So why do you add a direct declaration to this dependency, which is transitively included ? To explicitely declare that I use this dependency ?

Comment (adamretter) [reply pom.xml:82]: I'm not entirely sure... Transitive dependencies are very tricky. But I tend to find it you make the dependency plugin happy then you usually have less problems. Feel free to adjust my PR if you prefer; you should be able to push commits to my branch if you want to make some changes.
